test/use-location.test.js

```diff
@@ -45,6 +45,16 @@ describe("`value` first argument", () => {
     expect(result.current[0]).toBe("/");
     unmount();
   });
+
+  it("returns a pathname without a basepath", () => {
+    const { result, unmount } = renderHook(() =>
+      useLocation({ basepath: "/app" })
+    );
+
+    act(() => history.pushState(0, 0, "/app/dashboard"));
+    expect(result.current[0]).toBe("/dashboard");
+    unmount();
+  });
 });
 
```

Comment:
I would add a few extra test cases to handle the edge cases:
1. When the **basepath** is `/app` and current path is `/app` → `/`
2. When the **basepath** is `/app` and current path is `/home` → `/home`